libs/ui/src/tally_report_summary.test.tsx

```diff
@@ -55,7 +70,27 @@ test('Hides the other row in the voting method table when empty', () => {
     <TallyReportSummary
       totalBallotCount={3579}
       ballotCountsByVotingMethod={ballotCounts2}
+      election={electionSampleDefinition.election}
     />
   );
   expect(queryAllByText2('Other').length).toBe(0);
 });
+
+test('Is empty element if voting methods cannot be distinguished for election', () => {
+  mockOf(canDistinguishPrecinctAndAbsenteeBallots).mockReturnValue(false);
```

Comment:
Why does this need to be mocked here? My guess is that it's because `electionSampleDefinition.election` doesn't have `gridLayouts`. I think there's probably an election fixture that does have `gridLayouts` that you could use instead.

Reply:
Just isolating this component from the internal logic of the `canDistinguish` utility, since that could (in theory) change in the future

Reply:
Yep: https://github.com/votingworks/vxsuite/blob/e75f1d7d12e3e9bfdfa7ca43518ac07dbdaacfb5/libs/fixtures/src/data/electionGridLayoutNewHampshireHudson/election.json.ts.

Reply:
Ah gotcha. That's an interesting approach @kofi-q. On one hand it helps avoid testing the internal logic of `canDistinguish`, on the other hand the test now depends on internal implementation details of the component, which is something we've generally tried to avoid. I'm not sure which concern is more important

Reply:
Ohh that makes sense - yeah fine I'm either way, depending on where the team leans, philosophically. I think this is mostly residual habit from a previous job, so not tied to it - will switch over to varying the election data

Reply:
That actually reminds me that we did try to align a little while back on a testing philosophy. I'll send you the doc